dev-shipperhq/src/components/HomepageFeatures.js

```diff
@@ -32,7 +32,7 @@ const FeatureList = [
     Png: require('../../static/img/label-feature-comingsoon.png').default,
     description: (
       <>
-        Coming soon! Please contact sales@shipperhq.com to get early access (API available in beta).
+        Coming soon! Please <Link to="https://dev.shipperhq.com/contact">contact-us</Link> if you want to print Shipping Labels in your application! (Labels API available in beta).
```

Comment:
If you want to open in a new tab, use `<Link to="/contact" target="_blank" rel="noopener noreferrer">contact-us</Link> `. Are we sure we want to say "contact-us"? We use "contact us" everywhere else. 
```suggestion
        Coming soon! Please <Link to="/contact">contact-us</Link> if you want to print Shipping Labels in your application! (Labels API available in beta).
```